packages/salesforcedx-sobjects-faux-generator/src/generator/fauxClassGenerator.ts

```diff
@@ -81,8 +91,21 @@ export class FauxClassGenerator {
     ['complexvalue', 'Object']
   ]);
 
-  private static fieldName(decl: string): string {
-    return decl.substr(decl.indexOf(' ') + 1);
+  private static fieldDeclToString(decl: FieldDeclaration): string {
+    return `${FauxClassGenerator.commentToString(decl.comment)}${INDENT}${
+      decl.modifier
+    } ${decl.type} ${decl.name};`;
+  }
+
+  // VisibleForTesting
+  public static commentToString(comment?: string): string {
+    // for some reasons if the comment is on a single line the help context shows the last '*/'
+    return comment
+      ? `${INDENT}/* ${comment.replace(
+          /(\/\**\/)|(\/\**)|(\**\/)/g,
```

Comment:
One thing this may break is if you have a slash in your help text, e.g. "Bring a sweater and/or jacket" will become "Bring a sweater andor jacket".

Maybe this one instead
```suggestion
          /(\/\*+)|(\/\/)|(\*+\/)/g,
```

Reply:
yeah, that's a better regex 👍